### PR TITLE
Fix GH#21662: Add some missing instrument names

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #21662

Needs to come from the spreadsheet, hence only as a draft...

### CODE REVIEW

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -1052,17 +1052,13 @@
                   <genre>earlymusic</genre>
             </Instrument>
             <Instrument id="recorder">
+                  <init>soprano-recorder</init>
                   <family>recorders</family>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:1056]: I guess we can drop this line too and inherit via that `<init>`, but I'm not sure and don#t know how to check

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -13922,6 +13920,15 @@
                   <stafftype staffTypePreset="tab6StrFrench">tablature</stafftype>
                   <genre>earlymusic</genre>
             </Instrument>
+            <Instrument id="bass-viol">
+                  <init>viola-da-gamba</init>
+                  <family>viols</family>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:13925]: Same here

File: share/instruments/instruments.xml [hunk 3]
```diff
@@ -1052,17 +1052,13 @@
                   <genre>earlymusic</genre>
             </Instrument>
             <Instrument id="recorder">
+                  <init>soprano-recorder</init>
                   <family>recorders</family>
+                  <trackName>Recorder</trackName>
+                  <longName> Recorder</longName>
+                  <shortName>Rec.</shortName>
                   <description>Soprano or descant recorder, pitched in C.</description>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:1060]: we may be able to drop this one too.

File: share/instruments/instruments.xml [hunk 4]
```diff
@@ -1081,6 +1077,8 @@
                         <!--MIDI: Bank 0, Prog 74; MS General: Recorder-->
                         <program value="74"/> <!--Recorder-->
                   </Channel>
+                  <genre>common</genre>
+                  <genre>classroom</genre>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:1081]: IMHO Alto recorder belongs there too

File: share/instruments/instruments.xml [hunk 5]
```diff
@@ -13912,16 +13910,32 @@
                   </Channel>
                   <genre>earlymusic</genre>
             </Instrument>
+            <Instrument id="bass-viol">
+                <init>viola-da-gamba</init>
+                <family>viols</family>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:13915]: we may be able to drop this, but see above

**Review by shoogle — COMMENTED** [review]
See https://github.com/musescore/MuseScore/issues/21662#issuecomment-1966819582

**Comment by cbjeukendrup** [comment]
What is the situation with this PR? Would it be an idea to close it, make the genre changes in a separate PR (via the spreadsheet), and implement the `alias` system proposed by Peter in the future?

**Comment by Jojo-Schmitz** [comment]
I'm fine with closing it

**Pull request closed without merging**